shared/menubar/index.js

```diff
@@ -159,6 +164,7 @@ class Menubar extends Component {
       showMain={() => this.showMain()}
       showHelp={() => this.showHelp()}
       showUser={user => this.showUser(user)}
+      logIn={() => this.logIn()}
```

Comment:
There is a todo to show the login screen on main view if some other view is present in the main view... like signup?